demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -152,6 +152,7 @@ export function ProductsGrid(): React.ReactElement {
                         <IconButton component={StackLink} pageName="edit" payload={params.row.id}>
                             <Edit color="primary" />
                         </IconButton>
+
```

Comment:
Could we remove the empty line here? 😅

Reply:
Makes the generator code look a bit weird but yes 😄